services/blockchain-connector/shared/sdk/events.js

```diff
@@ -46,7 +47,10 @@ const subscribeToAllRegisteredEvents = async () => {
 			event,
 			async payload => {
 				// Force update nodeInfo cache on new chain events
-				if (event.startsWith('chain_')) await getNodeInfo(true);
+				if (event.startsWith('chain_')) {
+					await getNodeInfo(true);
```

Comment:
could we update the existing logic to force reload?

Incase, we need to update this for all chain events, then maybe we can remove the other listner?

Reply:
> could we update the [existing logic](https://github.com/LiskHQ/lisk-service/blob/feature/1157-prepare-lisk-service-v070-for-beta/services/blockchain-connector/shared/sdk/network.js#L37) to force reload?

Not necessary.

> Incase, we need to update this for all chain events, then maybe we can remove the other listner?
What do you mean by this?

Reply:
Yes, i had same concern. I'll verify and remove this logic